WordPress/Classes/Login/WordPressDotComAuthenticator.swift

```diff
@@ -17,7 +18,64 @@ struct WordPressDotComAuthenticator {
         case unknown(Swift.Error)
     }
 
+    @MainActor
+    func signIn(from viewController: UINavigationController) async {
+        let token: String
+        do {
+            token = try await authenticate(from: viewController)
```

Comment:
The animation on the top is distracting.

https://github.com/user-attachments/assets/3c36e695-af42-420b-a8a1-adb4c48e13c8

Reply:
I'm not sure why it looks like this for me. It's different on your recording. I think there is a bug with the first welcome page.

Reply:
🤔 I don't think there is an API for that. https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession

Reply:
Hmm I thought we had the ability to do it in the SwiftUI setup for self-hosted sites, but it's ok – we can dig into that further once we're not relying on a bunch of WPAuthenticator stuff anymore. 

I have a few suggestions about this in https://github.com/wordpress-mobile/WordPress-iOS/pull/23702

Reply:
> Hmm I thought we had the ability to do it in the SwiftUI setup for self-hosted sites

I just checked. The application password flow for self-hosted site also presents Safari view as a modal.